protocol/contracts/TempleZaps.sol

```diff
@@ -396,15 +408,22 @@ contract TempleZaps is Ownable {
     } else {
       // get impl contract as allowance target
 
+      // https://protocol.0x.org/en/latest/advanced/uniswap.html
+      //   > This function does not use allowances set on 0x. The msg.sender must have allowances set on Uniswap (or SushiSwap).
+      // 
+      // This has me confused, because the allowance required or you get an error.
```

Comment:
it's probably a documentation error. past simulations and tests have worked with setting allowance to `0x`. if you check the code tokens are transferred from caller directly to pair contracts.